### PR TITLE
Add back @theia/messages dependency to show toast messages

### PR DESCRIPTION
**Pull request by bhufmann**
This was accidentally removed during streamling of the example app. See commit 4c58c03

Signed-off-by: Bernd Hufmann <bernd.hufmann@ericsson.com>

### CODE REVIEW

File: theia-extensions/viewer-prototype/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     ],
     "dependencies": {
         "@theia/core": "1.45.1",
+        "@theia/messages": "1.45.1",
```
Comment (marcdumais-work) [inline theia-extensions/viewer-prototype/package.json:24]: optional: move it down according to alphabetical order?

**Review by marcdumais-work — APPROVED** [review]
LGTM - thanks Bernd! 

I added one comment about a minor optional change that could be done.

File: theia-extensions/viewer-prototype/package.json (hunk 1 above)
Comment (bhufmann) [reply theia-extensions/viewer-prototype/package.json]: Done

**Review by marcdumais-work — APPROVED** [review]
LGTM - thanks Bernd!

**Pull request merged**